test/Altinn.Platform.Events.Tests/TestingControllers/AppControllerTests.cs

```diff
@@ -413,7 +413,7 @@ public async void GetForOrg_ValidRequest_ReturnsListOfEventsAndNextUrl()
             {
                 // Arrange
                 string requestUri = $"{BasePath}/app/ttd/endring-av-navn-v2?from=2020-01-01Z&party=1337";
-                string expectedNext = $"https://platform.localhost:5080/events/api/v1/app/ttd/endring-av-navn-v2?after=e31dbb11-2208-4dda-a549-92a0db8c8808&from=2020-01-01Z&party=1337";
+                string expectedNext = $"http://localhost:5080/events/api/v1/app/ttd/endring-av-navn-v2?after=e31dbb11-2208-4dda-a549-92a0db8c8808&from=2020-01-01Z&party=1337";
```

Comment:
Not quite sure about this change here. Any particular reason you changed it? As requests are converted to the internal address using http when request hits the pod, we want to confirm that the scheme is still https then the next url is returned to the end user

Reply:
Not entirely correct. The code no longer set the scheme, but picks scheme from a config. If we want to test this specifically we could add gibberish to the config to confirm that the value is used.

Reply:
Had a chat with @b-brodie, we need to update environment values in the helm chart to reflect this change before this is merged then

Reply:
I've added values for the new `BaseUri` config, for all environments.